src/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@import "./utils/scripts";
+
+@import "./blocks/page";
+@import "./blocks/stop-watch.scss";
```

Comment:
The file extension .scss should be removed from the import statement. Use '@import "./blocks/stop-watch";' instead.